docs/user-guide/station-device-provisioning.md

```diff
@@ -0,0 +1,203 @@
+
+   1. e.g.: if you want to register a EU863 concentrator, not using CUPS, you should issue
+
+      ```powershell
+      .\loradeviceprovisioning.exe add --type concentrator --stationeui AABBCCFFFE001122 --region EU863 --no-cups
+      ```
+
+   2. e.g.: if you want to register a US902 concentrator, not using CUPS, which is expected to connect to LNS endpoint with client certificate, you should issue
+
+      ```powershell
+      .\loradeviceprovisioning.exe add --type concentrator --stationeui AABBCCFFFE001122 --region US902 --no-cups --client-certificate-thumbprint <AABBCCFFFE001122.crt Thumbprint Here>
+      ```
+
+   3. e.g.: if you want to register a EU863 concentrator, using CUPS, you should issue
+
+      ```powershell
+      .\loradeviceprovisioning.exe add --type concentrator --stationeui AABBCCFFFE001122 --region EU863 --client-certificate-thumbprint <AABBCCFFFE001122.crt Thumbprint Here> --certificate-bundle-location <path to AABBCCFFFE001122.bundle> --tc-uri wss://IP_OR_DNS:5001 --cups-uri https://IP_OR_DNS:5002
+      ```
+
+Please note that currently supported regions for the LoRa Device Provisioning CLI are EU863, US902, CN470RP1 and CN470RP2. Nevertheless, the tool is extensible and you can bring your own 'region.json' in the Cli-LoRa-Device-Provisioning\DefaultRouterConfig folder.
+
+## Manual configuration
+
+If you don't want to use the LoRa Device Provisioning CLI, in the following section we describe how to register an LBS in IoT Hub and how to store its configuration.
+
+1. Create an IoT Hub device that has a name equal to the LBS EUI in hex-representation, e.g. `AABBCCFFFE001122`. If you are running a dev kit on a Linux machine, the EUI will be retrieved from the MAC address of the eth0 interface as follows:
+
+    ```bash
+    cat /sys/class/net/eth0/address # prints the MAC Address of eth0
+    # Assuming aa:bb:cc:00:11:22 is the returned MAC Address
+    # your EUI will be AABBCCFFFE001122 
+    # Please note the insertion of the literals 'FFFE'  in the middle, as per https://doc.sm.tc/station/glossary.html?highlight=mac
+    ```
+
+2. The radio configuration needs to be stored as a desired twin property of the newly created LBS device. Make sure to store the configuration under `properties.desired.routerConfig`.
+   1. The configuration follows the `router_config` format from the LNS protocol as closely as possible. However, since device twins encode numbers as 32-bit values and given some configuration properties (such as EUIs) are 64-bit numbers, there are some minor differences.
+
+   2. The `JoinEui` nested array must consist of hexadecimal-encoded strings. The property should look similar to: `"JoinEui": [["DCA632FFFEB32FC5","DCA632FFFEB32FC7"]]`
+
+   3. A full configuration example might look like this, relative to the desired twin property path `properties.desired`:
+
+      <details>
+        <summary>EU863 Example Configuration</summary>
+
+        ```json
```

Comment:
it seems this is not required if you use the <details> tag:
![image](https://user-images.githubusercontent.com/1822123/146204112-e8e3c585-94c5-4c6c-8217-92f2f403a2ec.png)
```suggestion
```

Reply:
When not specifying it, formatting will be lost inside the details section. Therefore, will keep as it is

Reply:
interesting. It works here if I remove the ``` as well as the first and last bracket, and leave a newline. But then markdownlint is complaining 🙄
```
      <details>
        <summary>EU863 Example Configuration</summary>

          "routerConfig": {
```

Reply:
Let's keep it as it is if you don't mind 👍